internal/runners/clean/run_win.go

```diff
@@ -129,3 +132,44 @@ func removePaths(logFile string, paths ...string) error {
 
 	return nil
 }
+
+func checkAdmin() error {
```

Comment:
This either needs a comment or a more descriptive name (.. or both).

Reply:
Agreed. Naming is hard 😩